doc/Makefile.am

```diff
@@ -177,4 +160,33 @@ distclean-local: clean-local
 
 endif # HAVE_DOXYGEN
 
+install-data-hook:
+if HAVE_DOXYGEN
+	@if [ ! -d $(top_builddir)/doc/html ]; then \
+		echo ;\
+		echo "     No install data in '$(top_builddir)/doc/html'found! Please run 'make all' first." ;\
+		echo ;\
+		exit 1 ;\
+	fi
+	$(MKDIR_P) $(DESTDIR)$(htmldir)/html || exit 1
+	cp -a -f $(top_builddir)/doc/html $(DESTDIR)$(htmldir)
+	find $(DESTDIR)$(htmldir) -type f -name "*.md5" -exec rm {} \;
+endif # HAVE_DOXYGEN
+if BUILD_LICENSE_INSTALL
+	$(MKDIR_P) $(DESTDIR)$(docdir) || exit 1
+	$(INSTALL_DATA) $(top_srcdir)/LICENSE $(DESTDIR)$(docdir)
+	$(INSTALL_DATA) $(top_srcdir)/COPYING $(DESTDIR)$(docdir)
+	$(INSTALL_DATA) $(top_srcdir)/README $(DESTDIR)$(docdir)
+endif # BUILD_LICENSE_INSTALL
+
+uninstall-hook:
+if BUILD_LICENSE_INSTALL
```

Comment:
I thought about this because I tripped up over them not being uninstalled because I had re-run `configure`.  However, I think it right that `uninstall `should be the exact mirror of `install`.  As we may being building different packages for the different TLS libraries, some of them will have to be built with` ./configure --disable-license-install` to prevent clashes.  Doing a` make uninstall`  for ` ./configure --disable-license-install` should not affect the TLS library build that had ` ./configure --enable-license-install`

Reply:
Agreed.